### DISCUSSION TITLE
Kratos Migration failed in mainline version local build.

### DISCUSSION THREAD

**Discussion by Satish-Karunanithi** [ory/kratos#3677] [Q&A]
Hi, 
I am getting below error which i tried to execute migrate using the local build from mainline branch.
But it is successful when i pull the latest image from docker hub "oryd/kratos:latest". 
Could any one help in solving this issue? I am not sure if i am missing some thing on the latest mainline code build verison.

{"audience":"application","level":"info","migration_file":"migrations/sql/20230920171028000000_identity_search_index.postgres.up.sql","migration_name":"identity_search_index","msg":"Migration has not yet been applied, running migration.","service_name":"Ory Kratos","service_version":"","time":"2024-01-01T13:05:56.375121335Z","version":"20230920171028000000"}
{"audience":"application","level":"debug","msg":"0.0762 seconds","service_name":"Ory Kratos","service_version":"","time":"2024-01-01T13:05:56.39960271Z"}
Error: error executing migrations/sql/20230920171028000000_identity_search_index.postgres.up.sql, sql: CREATE EXTENSION IF NOT EXISTS pg_trgm;
CREATE EXTENSION IF NOT EXISTS btree_gin;

CREATE INDEX identity_credential_identifiers_nid_identifier_gin ON identity_credential_identifiers USING GIN (nid, identifier gin_trgm_ops);
: ERROR: data type uuid has no default operator class for access method "gin" (SQLSTATE 42704)
Usage:
  kratos migrate sql <database-url> [flags]

Flags:
  -c, --config strings   Path to one or more .json, .yaml, .yml, .toml config files. Values are loaded in the order provided, meaning that the last config file overwrites values from the previous config file.
  -h, --help             help for sql
  -e, --read-from-env    If set, reads the database connection string from the environment variable DSN or config file key dsn.
  -y, --yes              If set all confirmation requests are accepted without user interaction.

error executing migrations/sql/20230920171028000000_identity_search_index.postgres.up.sql, sql: CREATE EXTENSION IF NOT EXISTS pg_trgm;
CREATE EXTENSION IF NOT EXISTS btree_gin;

CREATE INDEX identity_credential_identifiers_nid_identifier_gin ON identity_credential_identifiers USING GIN (nid, identifier gin_trgm_ops);
: ERROR: data type uuid has no default operator class for access method "gin" (SQLSTATE 42704)

**Comment by piellarda** [comment]
This is what I'm seeing with the release 1.1.0 from today.

**Comment by alnr** [comment]
Check out this answer: https://dba.stackexchange.com/questions/165300/how-to-install-the-additional-module-pg-trgm

Reopen if this does not solve the issue.

**Comment by daominah** [comment]
I got the same error with docker postgres:10,
upgrading to postgres:16.4-bullseye solved the issue
